tika-core/src/test/java/org/apache/tika/mime/ProbabilisticMimeDetectionTest.java

```diff
@@ -135,7 +136,7 @@ private void testFile(String expected, String filename) throws IOException {
 
     private void testStream(String expected, String urlOrFileName, InputStream in)
             throws IOException {
-        assertNotNull("Test stream: [" + urlOrFileName + "] is null!", in);
+        assertNotNull(in, "Test stream: [" + urlOrFileName + "] is null!");
```

Comment:
This is a bug which we'll fix in main separately. Let's revert this here.

Reply:
Got it. Let me reorganize the commits and remove this now. The tests passed with no issues on my machine, though.

Reply:
Ha, right, the tests pass because our current test is "is the warning message not null", and in fact, the warning message is not null. LOL...